tokens/src/core/global/transition.json

```diff
@@ -1,7 +1,7 @@
 {
   "transition": {
-    "base": { "value": "all .2s ease-in-out", "type": "transition", "source": "$transition-base" },
-    "fade": { "value": "opacity .15s linear", "type": "transition", "source": "$transition-fade" },
-    "collapse": { "value": "height .35s ease", "type": "transition", "source": "$transition-collapse" }
+    "base": { "value": "all .2s ease-in-out", "type": "transition", "source": "$transition-base", "description": "Generic transition for any property change" },
+    "fade": { "value": "opacity .15s linear", "type": "transition", "source": "$transition-fade", "description": "Opacity transition that takes 150ms" },
+    "collapse": { "value": "height .35s ease", "type": "transition", "source": "$transition-collapse", "description": "Collapse transition that takes 350ms" }
```

Comment:
These raise the question of what is being described. Is the point of the description to describe the *value* or the *variable*?

"Generic transition for any property changed" seems to describe the *variable*, since it has no mention of "ease-in-out" or 200ms.

The other two seem to describe the *value*, directly mentioning the type of transition and amount of time.

I'd think the goal would be to describe the variable, so when someone sets a new value for it they'd know where it would be used. This would prevent the situation where someone updates the fade transition to be slower and the description still says "150ms"

Reply:
Totally agree with that, we should only describe the variable

Reply:
Left here because there can be more transitions and description should specify the difference between them so I assume that timings will be user-friendly here. Yes, they shouldn't be forgotten when those values will be changed in future.

Reply:
I think it's reasonable for now to include the value in this case. Can always revisit later. Generally, though, I agree the descriptions should be describing the intent of the variable, not so much the value itself.

The descriptions will be mostly useful to theme authors to get a better idea of what each design token's purpose is.